src/utils/listener/listeners.ts

```diff
@@ -179,7 +179,7 @@ export async function executeCheckSettingsChange(
     );
     vscode.commands.executeCommand(
       commands.setContext,
-      commands.isValidCredentials,
+      commands.isScanEnabled,
```

Comment:
you just renamed?
or use another parameter? - if use another - do we still need to save the old parameter somewhere?